### PR TITLE
Update update-codegen script

### PR DESCRIPTION
**Pull request by s4nji**
This PR fixes `hack/update-codegen.sh` script which currently generates its output correctly only if the project is in GOPATH.  
It will now output into a temporary directory and move it into the project directory.

### CODE REVIEW

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -20,16 +20,20 @@ set -o pipefail
...
-  --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate.go.txt \
+  --output-base "$TEMP" \
+  --go-header-file "$SCRIPT_ROOT/hack/boilerplate.go.txt" \
 
-# To use your own boilerplate text append:
-#   --go-header-file "${SCRIPT_ROOT}"/hack/custom-boilerplate.go.txt
+# Move generated code to project
+rm -r "./pkg/kubernetes/generated"
+mv "$TEMP/github.com/hellofresh/kangal/pkg/kubernetes/generated" "./pkg/kubernetes/generated"
+mv "$TEMP/github.com/hellofresh/kangal/pkg/kubernetes/apis/loadtest/v1/zz_generated.deepcopy.go" "./pkg/kubernetes/apis/loadtest/v1/zz_generated.deepcopy.go"
```
Comment (diegomarangoni) [inline hack/update-codegen.sh:39]:
```suggestion
cp -R $TEMP/github.com/hellofresh/kangal/pkg/kubernetes/generated/* ./pkg/kubernetes/generated/
cp -R $TEMP/github.com/hellofresh/kangal/pkg/kubernetes/apis/* ./pkg/kubernetes/apis/
```

Comment (s4nji) [reply hack/update-codegen.sh]: Updated 👍

File: hack/boilerplate.go.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/*
+Copyright The Kubernetes Authors.
```
Comment (diegomarangoni) [inline hack/boilerplate.go.txt:2]:
```suggestion
Copyright The Kangal Authors.
```

Comment (s4nji) [reply hack/boilerplate.go.txt:2]: Let's do this in a separate PR that also updates all the code-generated files

**Comment by s-radyuk** [comment]
+1

**Pull request merged**